CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ project(nvfuser)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+# This debug property lists the cmake target dependency graph.
+# set_property(GLOBAL PROPERTY GLOBAL_DEPENDS_DEBUG_MODE 1)
```

Comment:
nitpick: if we are going to keep it here, let's add a knob in `setup.py` to control it rather than having commented code.